Config/routes.js

```diff
@@ -7,6 +8,8 @@ const routes = (app) => {
     app.get('/dishes/:course', DishesController.getAllDishesOfType)
     app.post('/orders', OrdersController.createNewOrder)
     app.put('/orders/addToOrder', /* story 4 controller.function */ )
+    app.post('/orders', /* story 3 controller.function */ )
+    app.put('/orders/addToOrder', OrderController.addToOrder)
     app.delete('/orders', /* story 5 controller.function */ )
     app.put('/orders/editQuantity', /* story 6 controller.function */ )
     app.put('/orders/submitOrder', /* story 7 controller.function */ )
```

Comment:
You have some duplicate routes in here now, make sure to remove the old ones with the comments